ui/src/actions/beamline.js

```diff
@@ -46,30 +47,6 @@ export function updateBeamlineHardwareObjectStateAction(data) {
   return { type: BL_UPDATE_HARDWARE_OBJECT_STATE, data };
 }
 
-export function sendGetAllhardwareObjects() {
```

Comment:
From what I could tell, this action was not actually dispatched. The fetch call is duplicated and invoked directly in `general.js`.